src/App.css

```diff
@@ -4,18 +4,17 @@
 .App {
   text-align: center;
   display: grid;
-  grid-template-rows: 15vh auto 10vh;
+  grid-template-rows: 10vh auto 10vh;
   height: 100vh;
-  background: #fc0303;
+  background: #0c0d0d;
 }
 
 .App-Header {
   grid-row: 1/2;
+  border-radius: 5px;
   display: flex;
-  justify-content: center;
-  flex-direction: row;
   align-items: center;
-  border-radius: 5px;
```

Comment:
You deleting `border-radius` here and inserting `border-radius` again. Is this intended? Just pointing at it. Maybe a mistake?

Reply:
Deleted by mistake, yes. Then added back a few minutes later. That was confusing... sorry.